src/java.net.http/share/classes/module-info.java

```diff
@@ -25,7 +25,125 @@
 
 /**
  * Defines the HTTP Client and WebSocket APIs.
- *
+ * <p>
+ * <b id="httpclientprops">System properties used by the java.net.http API</b>
+ * <p>
+ * The following is a list of system networking properties used by the java.net.http
+ * client implementation in the JDK. Any properties below that take a numeric value
+ * assume the default value if given a string that does not parse as a number.
+ * Unless otherwise specified below, all values can be set in the {@code conf/net.properties}
+ * file. In all cases, values can be specified as system properties on the command line,
+ * in which case, any value in {@code conf/net.properties} is overridden. No guarantee is
+ * provided that property values can be set programatically with {@code System.setProperty()}.
+ * Other implementations of this API may choose not to support these properties.
+ * <ul>
+ * <li><p><b>{@systemProperty jdk.httpclient.allowRestrictedHeaders}</b> (default: see below)<br>
+ * A comma-separated list of normally restricted HTTP header names that users may set in HTTP requests
+ * or by user code in HttpRequest instances. By default, the following request headers are not allowed
+ * to be set by user code: connection, content-length, expect, host, and upgrade. You can override this
+ * behavior with this property. The names are case-insensitive and whitespace is ignored. Note that this
+ * property is intended for testing and not for real-world deployments. Protocol errors or other undefined
+ * behavior are likely to occur when using this property. There may be other headers that are restricted
+ * from being set depending on the context. This includes the "Authorization" header when the relevant
+ * HttpClient has an authenticator set. These restrictions cannot be overridden by this property.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.bufsize}</b> (default: 16384 bytes or 16 kB)<br>
+ * The size to use for internal allocated buffers in bytes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.connectionPoolSize}</b> (default: 0)<br>
+ * The maximum number of connections to keep in the HTTP/1.1 keep alive cache. A value of 0 means that
+ * the cache is unbounded.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.connectionWindowSize}</b> (default: 2^26)<br>
+ * The HTTP/2 client connection window size in bytes. The maximum size is 2^31-1. This value cannot be
+ * smaller than the stream window size.
```

Comment:
Should this additionally point to the `jdk.httpclient.windowsize` system property? Something like `This value cannot be smaller than the stream window size, which can be configured through {@code jdk.httpclient.windowsize} system property`?

Reply:
Good idea!